KadaneAlgorithm.cpp

```diff
@@ -0,0 +1,27 @@
+#include<bits/stdc++.h>
```

Comment:
Kindly use iostream and while dealing with console operations add a std:: in front, like std::cin << , std::cout << .